tests/unit/plugins/modules/test_pacemaker_resource.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Author: Dexter Le (dextersydney2001@gmail.com)
+# Largely adapted from test_redhat_subscription by
+# Jiri Hnidek (jhnidek@redhat.com)
+#
+# Copyright (c) Dexter Le (dextersydney2001@gmail.com)
+# Copyright (c) Jiri Hnidek (jhnidek@redhat.com)
+#
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+from ansible_collections.community.general.plugins.modules import pacemaker_resource
+from .helper import Helper, RunCommandMock
+
+
+Helper.from_module(pacemaker_resource, __name__, mocks=[RunCommandMock])
```

Comment:
After you rebase your branch, the tests will fail. You will then need to change these lines to:
```suggestion
from .uthelper import UTHelper, RunCommandMock


UTHelper.from_module(pacemaker_resource, __name__, mocks=[RunCommandMock])
```